airflow/task/task_runner/standard_task_runner.py

```diff
@@ -186,3 +193,12 @@ def get_process_pid(self) -> int:
         if self.process is None:
             raise RuntimeError("Process is not started yet")
         return self.process.pid
+
+    def _read_task_utilization(self):
+        while True:
+            dag_id = self._task_instance.dag_id
+            task_id = self._task_instance.task_id
+            mem_usage = self.process.memory_percent()
+            cpu_usage = self.process.cpu_percent(interval=1)
+            Stats.gauge(f"task.mem_usage_percent.{dag_id}.{task_id}", mem_usage)
+            Stats.gauge(f"task.cpu_usage_percent.{dag_id}.{task_id}", cpu_usage)
```

Comment:
does this CPU utilization for one core or all cores?

Reply:
All cores. If you want for one core, you can set `percpu` to `True`. See [doc](https://psutil.readthedocs.io/en/latest/#psutil.cpu_percent)

Reply:
Same as a [`top`](https://man7.org/linux/man-pages/man1/top.1.html) output, you could have 500% utilisation in case if you have 5+ cores and process use threads which spread across different cores